.github/workflows/checksums.yml

```diff
@@ -0,0 +1,57 @@
+name: FIPS Checksums
+on: [pull_request]
+jobs:
+  apply-label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: install unifdef
+        run: |
+            sudo apt-get update
+            sudo apt-get -yq --no-install-suggests --no-install-recommends --force-yes install unifdef
+      - uses: actions/checkout@v2
+        with:
+          ref: master
```

Comment:
I thought we wanted to diff against the branch point not master.

A bit of search and tweaking produced this to get the point.
`diff -u <(git rev-list --first-parent `git branch --show-current`) <(git rev-list --first-parent master) | sed -ne 's/^ //p' | head -1`
I've no idea if this can be integrated.  Perhaps in a later _run_ action?

Reply:
The description sounds like what `git merge-base` does, but I did not try to decipher what your version actually does.

Reply:
That's much much easier:
```
git merge-base --fork-point master
```

Reply:
There might be a different way how to find out the merge branch of a PR. I'll try to find it.

Reply:
Fixup pushed